### ISSUE TITLE
Feature request - keep /var/cache/xbps/ cleaner!

### ISSUE THREAD

**Issue by guayaseal** [void-linux/xbps#529]
Something like 'pacman -Sc' in Arch:
https://archlinux.org/pacman/pacman.8.html#_sync_options_apply_to_em_s_em_a_id_so_a
would be welcome

**Comment by Vaelatern** [comment]
`xbps-remove -O`

**Comment by guayaseal** [comment]
No! If I install a package then uninstall it then "xbps-remove -O" won't remove the package file. If I install, let's say, KDE and GNOME then remove them and decide to install XFCE I will still have several GBs of xbps files for KDE and GNOME. "xbps-remove -O"  won't clean them up!

**Comment by fvalasiad** [comment]
> No! If I install a package then uninstall it then "xbps-remove -O" won't remove the package file. If I install, let's say, KDE and GNOME then remove them and decide to install XFCE I will still have several GBs of xbps files for KDE and GNOME. "xbps-remove -O" won't clean them up!

`xbps-remove -o`

**Comment by Chocimier** [comment]
`xbps-remove -o` uninstall packages, an issue is about 

> Remove [compressed, unextracted] packages that are no longer installed from the cache

**Comment by guayaseal** [comment]
?
https://man.voidlinux.org/xbps-remove.1 says:

-o, --remove-orphans
Removes installed package orphans that were installed automatically (as dependencies) and are not currently dependencies of any installed package. See --mode auto in xbps-pkgdb(1)

**Comment by classabbyamp** [comment]
#530
